### ISSUE TITLE
🆕 Software Suggestion | Sanipasse

### ISSUE THREAD

**Issue by lovasoa** [privacytools/privacytools.io#2301]
## Basic Information

**Name:** Sanipasse
**Category:** Health
**URL:** https://sanipasse.fr

## Description

Sanipasse is a privacy-preserving open-source alternative to the french government's health pass checker.

## Why I am making the suggestion

Here is an article about the why and how of sanipasse: https://sanipasse.fr/french-health-pass

In short: France has adopted a law that makes test or vaccine certificates mandatory to attend large events.
The official application to check these certificates is closed-source, and is not even publicly available. It requires disclosing private health information to event organizers. 

Sanipasse is an alternative open-source web application that allows:
 - reading health certificates and checking for their authenticity
 - organizing events with a mandatory health pass where guests can validate their passes themselves without disclosing them to organizers.

## My connection with the software

I am the author.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by lrq3000** [comment]
Amazing app, thank you so much for making it!

Here is the source code for those interested, under AGPLv3: https://github.com/lovasoa/sanipasse

Unfortunately, I think it's unlikely it gets added to PTIO since there is no section for country-specific apps, but if we had it sure would fit!

Also, can the app be used offline, eg, by downloading an archive from github? It would be nice if you could make such a release so people can easily download it and run offline.

**Comment by lovasoa** [comment]
Yes, it has a service worker with a cache, so the online version should work offline after the first load. I haven't taken the time to test that properly yet, though. The webapp is also installable from chrome desktop, chrome mobile, and firefox mobile. 

If you want to run the server part locally, you can git clone it, then install dependencies with npm install, and run it in development mode with npm run dev.

**Comment by ph00lt0** [comment]
I think it would be worth it to add these apps in a separate section listing FOSS alternatives. For the UK I saw this app popping up in f-droid:
https://f-droid.org/en/packages/covidsecure.uk.venuecheckin/

In many countries we see that access to venue's is being restricted exclusively to people using apps. Whatever we think of this, it would be good to provide people with the least invasive ways to go to a venue.

**Comment by lrq3000** [comment]
Ok then where on PTIO (what section)?

**Comment by ph00lt0** [comment]
@lrq3000 software/covid-19 ?

**Comment by freddy-m** [comment]
@lrq3000 @ph00lt0 do you want to open a seperate issue for covid-19 related apps? @privacytools/editorial do you think this would fit the site? I personally have nothing against it...

**Comment by lrq3000** [comment]
@freddy-m if the editorial board is interested, yes I can and I can make a PR too.

**Comment by freddy-m** [comment]
@lrq3000 certainly make an issue - we'll decide as to whether we go ahead with the PR.

**Comment by lrq3000** [comment]
For information, Sanipasse now supports european Digital Covid Certificate (DCC), although the norm is changing so support may break in the future: https://github.com/lovasoa/sanipasse/issues/9

**Comment by lovasoa** [comment]
We will keep supporting the european DGC

**Comment by lrq3000** [comment]
Thank you and your colleagues for your great work @lovasoa !
